styles/h5p-tooltip.css

```diff
@@ -0,0 +1,33 @@
+.h5p-tooltip {
+	display: none;
+	position: absolute;
+	bottom: 100%;
+	left: 50%;
+	transform: translateX(-50%);
+
+	z-index: 3; /* Must be above feedback, which is 2 */
+
+	font-size: 0.75em;
+	line-height: 1.5em;
+
+	padding: 0 0.3em;
+	white-space: nowrap;
+
+	background: #000;
+	color: #FFF;
+}
+  
+.h5p-tooltip-visible {
+	display: block;
+}
+
+.h5p-tooltip-right {
+	left: auto;
+	right: 0;
+	transform: translateX(0);
+}
+
+.h5p-tooltip-left {
+	left: 0;
+	transform: translateX(0);
+}
```

Comment:
It seems to be something wrong with the indentation here. We should use 2 spaces.